app/views/chargings/_form_charging.html.slim

```diff
@@ -0,0 +1,18 @@
+        = form_with(url: car_chargings_path(car_id: @car.id), method: :create) do |form|
+          p[class="h4" id="chargingFormHeader"] Add Charging          
+          div.f
+            = form.text_field :brand_station, autofocus: true, :size=>"50", placeholder: "Brand Station",autocomplete: "on", class: "col-sm-6 col-form-label"
+
+          div.f    
+            = form.number_field :amount, required: true, autofocus: true, placeholder: "Amount",autocomplete: "off", class: "col-sm-6 col-form-label"
+
+          div.f
+            = form.date_field :date, required: true, autofocus: true, autocomplete: "off", class: "col-sm-6 col-form-label", required: true
+
+          div.f
+            = form.hidden_field :id, name: "id", required: false
```

Comment:
You don't need this field, since you're not setting anything there, it's just empty
<img width="1257" alt="Screenshot 2022-09-05 at 16 26 35" src="https://user-images.githubusercontent.com/88270887/188471588-7e2d0267-187e-4559-b5f1-320345cb3eeb.png">

Reply:
I'm using it on :update